setup.py

```diff
@@ -48,8 +48,8 @@
     python_requires=">=3.8",
     install_requires=[
         "marshmallow>=3.18.0,",
-        "typing-inspect~=0.9.0",
-        "typeguard~=4.0.0",
+        "typing-inspect>=0.9.0",
+        "typeguard>=4.0.0",
```

Comment:
Do we want to totally remove the upper bound on typeguard?  Or would  `typeguard~=4.0` (or equivalently `typeguard>=4.0,<5` be safer?

Typeguard seems to follow semantic versioning.  Backward-incompatible API changes were introduced in `typeguard==4.0.0rc1` and `typeguard==3.0.0`.  (Changlog [here](https://typeguard.readthedocs.io/en/latest/versionhistory.html).)

Reply:
> Do we want to totally remove the upper bound on typeguard? Or would `typeguard~=4.0` (or equivalently `typeguard>=4.0,<5` be safer?
> 
> Typeguard seems to follow semantic versioning. Backward-incompatible API changes were introduced in `typeguard==4.0.0rc1` and `typeguard==3.0.0`. (Changlog [here](https://typeguard.readthedocs.io/en/latest/versionhistory.html).)

I have added the upper bound on `typeguard`